mindjunkies/forums/tests/test_views.py

```
@@ -29,58 +27,82 @@
+            course_description="Detailed course description",
+            teacher=self.user,
+            level="beginner",
+        )
+
+    def test_get(self):
+        """Test ForumHomeView renders correctly"""
+        request = self.factory.get("/")
+        request.user = self.user
+        response = ForumHomeView.as_view()(request, course_slug=self.course.slug)
+        self.assertEqual(response.status_code, 200)
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)